README.md

```diff
@@ -17,7 +17,7 @@ Before running `backups2datalad`, the following setup must be performed:
   required.
 
 - [git-annex](https://git-annex.branchable.com) must be installed.  At least
-  version 10.20230126 is required, though you should endeavor to obtain the
+  version 10.20240227 is required, though you should endeavor to obtain the
```

Comment:
would you be so kind to add explicit check for git-annex version, e.g. 

```shell
❯ python -c 'from datalad.support.external_versions import external_versions as externals; print(externals["cmd:annex"] >= "10.20240227")'
False
❯ git annex version
git-annex version: 10.20240129
```

so we do not need to deal with aftermath of potential running with outdated ones and crashing only later on, requiring cleaning etc

Reply:
What does that code snippet do if git-annex isn't installed?  Would `externals["cmd:annex"]` be `None` or absent?

Reply:
```shell
❯ python -c 'from datalad.support.external_versions import external_versions as externals; print(repr(externals["cmd:annex"]))'
LooseVersion ('10.20240129')
❯ sudo dpkg --purge --force-depends git-annex
...
❯ python -c 'from datalad.support.external_versions import external_versions as externals; print(repr(externals["cmd:annex"]))'
None
```

Reply:
Done: c81b62b